packages/client/src/components/Card/Card.module.scss

```
@@ -0,0 +1,7 @@
+@import '@styles/variables';
+
+.card {
+  border-radius: $cardBorderRadius;
+  background: $cardBackgroundColor;
+  box-shadow: $cardShadow !important;
```

Рекомендую группировать по css свойству, как [тут](https://paradigm.mail.ru/design_tokens/colors):
```suggestion
  border-radius: $borderRadiusCard;
  background: $backgroundColorCard;
  box-shadow: $shadowCard !important;
```